scan/main.py

```diff
@@ -52,8 +57,23 @@ def run(self):
             memory=True,  # Enable memory usage for enhanced task execution
         )
 
-        result = _run_crew(crew)
-        return result
+        results = _run_crew(crew)
+        logger.info(f"Raw results: {results}")
```

Comment:
This log will always show, is that what you want? Or do you just want it for development/debugging? If the latter then you can add a `verbose` parameter to the `run` function that takes a boolean, then only show the log if `verbose` is `True`. You would also want to add a command line flag to the running of the program to specify running in verbose mode.